fe/fe-core/src/test/java/com/starrocks/connector/analyzer/SimpleQueryAnalyzer.java

```diff
@@ -333,28 +333,30 @@ public Void visitTableFunction(TableFunctionRelation node, Void scope) {
             node.setTableFunction(tableFunction);
             node.setChildExpressions(node.getFunctionParams().exprs());
 
-            if (node.getColumnNames() == null) {
+            if (node.getColumnOutputNames() == null) {
                 if (tableFunction.getFunctionName().getFunction().equals("unnest")) {
                     // If the unnest variadic function does not explicitly specify column name,
                     // all column names are `unnest`. This refers to the return column name of postgresql.
                     List<String> columnNames = new ArrayList<>();
                     for (int i = 0; i < tableFunction.getTableFnReturnTypes().size(); ++i) {
                         columnNames.add("unnest");
                     }
-                    node.setColumnNames(columnNames);
+                    node.setColumnOutputNames(columnNames);
                 } else {
-                    node.setColumnNames(new ArrayList<>(tableFunction.getDefaultColumnNames()));
+                    node.setColumnOutputNames(new ArrayList<>(tableFunction.getDefaultColumnNames()));
                 }
             } else {
-                if (node.getColumnNames().size() != tableFunction.getTableFnReturnTypes().size()) {
+                if (node.getColumnOutputNames().size() != tableFunction.getTableFnReturnTypes().size()) {
                     throw new SemanticException("table %s has %s columns available but %s columns specified",
-                            node.getAlias().getTbl(), node.getColumnNames().size(), tableFunction.getTableFnReturnTypes().size());
+                            node.getAlias().getTbl(),
+                            node.getColumnOutputNames().size(),
+                            tableFunction.getTableFnReturnTypes().size());
```

Comment:
available columns is `tableFunction.getTableFnReturnTypes()` and specified columns is `node.getColumnOutputNames()` ?